src/renderer/components/Header/Header.tsx

```diff
@@ -39,6 +40,7 @@ export const Header = memo(() => {
                     >
                         <SystemStats />
                         <DependencyManagerButton />
+                        <KoFiButton />
                         <SettingsButton />
```

Comment:
I wanted setting to be the rightmost one, and the dependency manager to be next to the system stats still. I could move it though, where would you rather it be

Reply:
No, that works for me. TBH, the button doesn't have a logical place anywhere since it serves no functional purpose. As long it's not in the way, it's fine.